command/proc_group_linux.go

```diff
@@ -0,0 +1,13 @@
+// +build linux
```

Comment:
Because the file is postfixed with _linux.go, it will only build on linux, no need for this build tag.